### PR TITLE
add comments for ops LastLevelMaxPool to avoid potential confusion

### PR DESCRIPTION
**Pull request by iaoqian**
This PR added some comments for ops LastLevelMaxPool which not really applied a max_pool2d to input but did subsampling actually. These comments attempt to clarify that and avoid potential confusion in future.

### CODE REVIEW

**Comment by vfdev-5** [comment]
Thanks for the PR @iaoqian , however, checking Detectron2 code, they also have `kernel_size=1`: 
https://github.com/facebookresearch/detectron2/blob/d4a5f28e01b2babbaba9f90198fb95f5c661cccd/detectron2/modeling/backbone/fpn.py#L200

and same for Detectron implementation: 
https://github.com/facebookresearch/Detectron/blob/6020e9a5aaaa21316cb87a8bc89605e2594310e7/detectron/modeling/FPN.py#L229

Seems like it is intended to have kernel_size=1 as they left a comment:
> # Use max pooling to simulate stride 2 subsampling

**Comment by NicolasHug** [comment]
Also from the [paper](https://arxiv.org/pdf/1612.03144.pdf):

image

**Comment by iaoqian** [comment]
Well, Thanks @vfdev-5 @NicolasHug for your reply. That makes sense. In this way, LastLevelMaxPool may not be a proper name, I think. Maybe we can refactor this class with name 'LastLevelSubsampling' or something to make it more reasonable. But, anyway, closing this PR.

**Comment by NicolasHug** [comment]
I agree with you @iaoqian , the current naming might be a bit misleading. Unfortunately we can't really change it because it's public and it would require a deprecation (not sure that it's really worth it). But thanks for sending this PR and for raising this to our attention.
What we can do to avoid such confusion in the future is to clarify in the docstring that the code is intended as-is (i.e. that the Module is just doing subsampling, not max-pooling). Perhaps we can re-purpose your PR for that?

**Comment by iaoqian** [comment]
I understand @NicolasHug. I think we can repurpose this PR by adding some code comments to avoid this issue.

File: torchvision/ops/feature_pyramid_network.py [hunk 1]
```diff
@@ -206,7 +206,7 @@ def forward(self, x: Dict[str, Tensor]) -> Dict[str, Tensor]:
 
 class LastLevelMaxPool(ExtraFPNBlock):
     """
-    Applies a max_pool2d on top of the last feature map
+    Applies a max_pool2d (not actual max_pool2d) on top of the last feature map
```
Comment (NicolasHug) [inline torchvision/ops/feature_pyramid_network.py:209]:
```suggestion
    Applies a max_pool2d (not actual max_pool2d, we just subsample) on top of the last feature map
```

**Review by NicolasHug — APPROVED** [review]
Thank you @iaoqian !

**Pull request merged**